test/query_members.js

```diff
@@ -59,124 +56,148 @@ describe('Query Members', function() {
 		await channel.addMembers([rob2]);
 		await channel.addMembers([adam]);
 		await channel.addMembers([banned]);
-
 		await channel.inviteMembers([invited, pending, rejected]);
 
 		// mod bans user banned
 		await channel.banUser(banned, { user_id: mod });
 
-		//accept the invite
+		// accept the invite
 		const clientA = await getTestClientForUser(invited);
 		await clientA.channel('messaging', channel.id).acceptInvite();
 
-		//reject the invite
+		// reject the invite
 		const clientR = await getTestClientForUser(rejected);
 		await clientR.channel('messaging', channel.id).rejectInvite();
 	});
 
+	it('query members with multiple filters client side', async function() {
```

Comment:
@thesyncim this query should pass according to my understanding but fails. How we build query with `and` and `or` might be wrong. Anything wrong on syntax ? Otherwise, I will see SQL side.

Reply:
It was in backend, fixed.